doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,298 @@
+   :functions: intel_gvt_hypervisor_read_gpa
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_write_gpa
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_gfn_to_pfn
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_map_gfn_to_mfn
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_set_trap_area
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_set_pvmmio
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hyperisor_pause_domain
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_unpause_domain
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mpt.h
+   :functions: intel_gvt_hypervisor_dom0_ready
+
+
+GVT-g intel_gvt_ops interface
+*****************************
+
+This section contains APIs for GVT-g intel_gvt_ops interface. Sources are found
+in the `ACRN kernel GitHub repo`_
+
+
+.. _ACRN kernel GitHub repo: https://github.com/projectacrn/acrn-kernel/
+
+
+.. code-block:: c
+
+	static const struct intel_gvt_ops intel_gvt_ops = {
+		.emulate_cfg_read = intel_vgpu_emulate_cfg_read,
+		.emulate_cfg_write = intel_vgpu_emulate_cfg_write,
+		.emulate_mmio_read = intel_vgpu_emulate_mmio_read,
+		.emulate_mmio_write = intel_vgpu_emulate_mmio_write,
+		.vgpu_create = intel_gvt_create_vgpu,
+		.vgpu_destroy = intel_gvt_destroy_vgpu,
+		.vgpu_reset = intel_gvt_reset_vgpu,
+		.vgpu_activate = intel_gvt_activate_vgpu,
+		.vgpu_deactivate = intel_gvt_deactivate_vgpu,
+	};
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/cfg_space.c
+   :functions: intel_vgpu_emulate_cfg_read
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/cfg_space.c
+   :functions: intel_vgpu_emulate_cfg_write
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mmio.c
+   :functions: intel_vgpu_emulate_mmio_read
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/mmio.c
+   :functions: intel_vgpu_emulate_mmio_write
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/vgpu.c
+   :functions: intel_gvt_create_vgpu
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/vgpu.c
+   :functions: intel_gvt_destroy_vgpu
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/vgpu.c
+   :functions: intel_gvt_reset_vgpu
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/vgpu.c
+   :functions: intel_gvt_activate_vgpu
+
+.. kernel-doc:: drivers/gpu/drm/i915/gvt/vgpu.c
+   :functions: intel_gvt_deactivate_vgpu
+
+ACRN-GT sysfs interface
+***********************
+
+This section contains APIs for the ACRN-GT sysfs interface. Sources are found
+in the `ACRN kernel GitHub repo`_
+
+
+sysfs nodes
+===========
+
+In below examples all accesses to these interfaces are via bash command
+``echo`` or ``cat``. This is a quick and easy way to get/control things. But
+when these operations fails, it is impossible to get respective error code by
+this way.
+
+When accessing sysfs entries, people should use library functions like
```

Comment:
change ``like`` to ``such as``

``like`` should be used for comparison (it floats like a butterfly) while ``such as`` is used for examples (I prefer colors such as red and blue).

Reply:
updated.